waggle-dance-core/src/main/java/com/hotels/bdp/waggledance/mapping/service/impl/PrefixBasedDatabaseMappingService.java

```diff
@@ -80,32 +81,40 @@ public PrefixBasedDatabaseMappingService(
     init(initialMetastores);
   }
 
-  private void init(List<AbstractMetaStore> federatedMetaStores) {
+  private void init(List<AbstractMetaStore> abstractMetaStores) {
```

Comment:
I don't think you need to rename this variable but if you do, name it `metastores` and subsequent usage should be `metastore` instead of `abstractMetastore`